app/lib/submission_builder/ty2022/states/az/documents/state1099_g.rb

```diff
@@ -10,8 +10,17 @@ def document
               form1099g = @kwargs[:form1099g]
 
               build_xml_doc("State1099G", documentId: "State1099G-#{form1099g.id}") do |xml|
-                xml.PayerName payerNameControl: form1099g.default_payer_name.gsub(/\s+/, '').upcase[0..3] do
-                  xml.BusinessNameLine1Txt form1099g.default_payer_name
+                if form1099g.payer_name && form1099g.payer_name != ''
+                  xml.PayerName payerNameControl: form1099g.payer_name.gsub(/\s+/, '').upcase[0..3] do
+                    xml.BusinessNameLine1Txt form1099g.payer_name
+                  end
```

Comment:
Does this remove sspaces and capitalize the first four letters? Might be worth checking with an input payer name in the output XML and perhaps edit this if desired.